package.json

```diff
@@ -51,7 +51,7 @@
     "eslint-plugin-prettier": "^3.4.0",
     "husky": ">=6",
     "jest": "^27.0.6",
-    "lerna": "^4.0.0",
+    "lerna": "6.4.1",
```

Comment:
any reason to keep it at specific version vs minimum version?

Reply:
In lerna 6.5.0 the structure of the package.json was changed which breaks our commitlint config. There are open issues on lerna's Github that address this so we will wait for the fix then we can upgrade/use minimum version.